extensions/amp-position-observer/0.1/amp-position-observer.js

```diff
@@ -353,9 +353,8 @@ export class AmpVisibilityObserver extends AMP.BaseElement {
     let scene;
     if (this.targetId_) {
       scene = user().assertElement(
-        // TODO: In friendly iframe, getAmpDoc() incorrectly fetches the parent
-        // doc. Need to change it to return the FIE AMP doc.
-        this.getAmpDoc().getElementById(this.targetId_),
+        // TODO: Revert to this.getAmpDoc once AMP FIE is launched.
```

Comment:
pls reference to `ampdoc-fie` experiment so that later this will be visible via code search while the experiment is being cleaned up.